cudadeprecation 14&15 (#326)
rapidsai/docs: _notices/rsn0027.md

---
layout: notice
parent: RAPIDS Support Notices
grand_parent: RAPIDS Notices
nav_exclude: true
notice_type: rsn
# Update meta-data for notice
notice_id: 27 # should match notice number
notice_pin: true # set to true to pin to notice page
title: "Deprecation announcement for CUDA 11.4 and 11.5 containers"
notice_author: RAPIDS Ops
notice_status: In Progess
notice_status_color: yellow
# 'notice_status' and 'notice_status_color' combinations:
#   "Proposal" - "blue"
#   "Completed" - "green"
#   "Review" - "purple"
#   "In Progress" - "yellow"
#   "Closed" - "red"
notice_topic: Platform Support Change

> notice_rapids_version: "v23.02+"

Reviewer: ```suggestion
notice_rapids_version: "v23.04+"
```

> notice_created: 2023-01-27

Reviewer: ```suggestion
notice_created: 2023-03-08
```

# 'notice_updated' should match 'notice_created' until an update is made

> notice_updated: 2023-01-27

Reviewer: ```suggestion
notice_updated: 2023-03-08
```

---

## Overview

> RAPIDS is deprecating `CUDA 11.4` and `CUDA 11.5` containers in its upcoming `v23.02` release. RAPIDS will plan to EOL these containers in a future release (currently targeting `v23.04`), at which point `CUDA 11.4` and `CUDA 11.5` containers will no longer be published.

Reviewer: ```suggestion
RAPIDS is deprecating `CUDA 11.4` and `CUDA 11.5` containers effective its `v23.04` release. RAPIDS will plan to EOL these containers in a future release (currently targeting `v23.06`), at which point `CUDA 11.4` and `CUDA 11.5` containers will no longer be published.
```

## Impact

> Once these containers reach EOL (currently targeting `v23.04` release), RAPIDS will no longer publish these containers. Conda packages will still support `CUDA 11.4` and `CUDA 11.5` due to CUDA Enhanced Compatibility.

Reviewer: ```suggestion
Once these containers reach EOL (currently targeting `v23.06` release), RAPIDS will no longer publish these containers. Note that conda packages will still support `CUDA 11.4` and `CUDA 11.5` due to CUDA Enhanced Compatibility.
```
